tests/test_scylla_relocatable_cluster.py

```diff
@@ -65,3 +65,12 @@ def test_cqlsh(self, relocatable_cluster):
         for s in ['(1 rows)', 'key', '1']:
             assert s in rv[0]
         assert rv[1] == ''
+
+
+    def test_get_default_scylla_yaml(self, relocatable_cluster):
+        install_dir = relocatable_cluster.get_install_dir()
+        scylla_yaml = get_default_scylla_yaml(install_dir)
+        assert 'enable_tablets' not in scylla_yaml
```

Comment:
i'd use a more stable option in this file, as in new reloc packages, this does not hold anymore.

Reply:
fixed